bigchaindb/consensus.py

```diff
@@ -1,4 +1,11 @@
+import logging
+
 from bigchaindb.util import verify_vote_signature
+from bigchaindb.common.schema import SchemaValidationError, \
+    validate_vote_schema
```

Comment:
Usually we prefer:

```python
from ... import (
    a,
    b,
    ...
)
```

(See [our style guide](https://github.com/bigchaindb/bigchaindb/blob/master/PYTHON_STYLE_GUIDE.md#how-to-format-long-import-statements))